.gitignore

```diff
@@ -1,2 +1,4 @@
+.DS_Store
 node_modules/
 .vdiff/
+.d2l-test/
```

Comment:
I guess every repo that uses this will need to add this ignore entry?

Reply:
No, it'll only exist in CI on test jobs. I just forgot to stop it from always writing the file because I was testing locally. I'll do that now.